clients/venice-client/src/main/java/com/linkedin/venice/fastclient/meta/LeastLoadedClientRoutingStrategy.java

```diff
@@ -27,19 +26,7 @@ public String getReplicas(long requestId, int groupId, List<String> replicas) {
     if (replicas.isEmpty()) {
       return null;
     }
-    List<String> availReplicas = new ArrayList<>();
-    /**
-     * For even distribution, we need to shuffle the replicas.
-     */
-    Collections.shuffle(replicas);
```

Comment:
We cannot remove this shuffle. This is important to achieve even distribution when all replicas have the same pending requests. Usually this happens when pending requests are all 0.